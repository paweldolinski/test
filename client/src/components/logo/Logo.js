import React from "react";
import Img from "../../assets/img/logo2.svg";

const Logo = () => {
  return (
    <div className="logo">
      <img className="logo__logo" src={Img} alt="logo" />
    </div>
  );
};

export default Logo;
