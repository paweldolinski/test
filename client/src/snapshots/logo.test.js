

import React from 'react';
import Logo from '../components/logo/Logo';
import renderer from 'react-test-renderer';

it('logo renders correctly', () => {
  const tree = renderer
    .create(<Logo />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});














