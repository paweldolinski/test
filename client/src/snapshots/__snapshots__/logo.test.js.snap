// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo renders correctly 1`] = `
<div
  className="logo"
>
  <img
    alt="logo"
    className="logo__logo"
    src="logo2.svg"
  />
</div>
`;
